CRM/Upgrade/Incremental/php/FiveSixtyFour.php

```diff
@@ -33,9 +33,14 @@ public function upgrade_5_64_alpha1($rev): void {
     $this->addTask('Drop unused civicrm_action_mapping table', 'dropTable', 'civicrm_action_mapping');
     $this->addTask('Update post_URL/cancel_URL in logging tables', 'updateLogging');
     $this->addTask('Add in Everybody ACL Role option value', 'addEveryBodyAclOptionValue');
-    $this->addTask('Fix double json encoding of accepted_credit_cards field in payment processor table', 'fixDoubleEscapingPaymentProcessorCreditCards');
```

Comment:
@eileenmcnaughton this should be kept iirc

Reply:
argh